books/kestrel/crypto/r1cs/sparse/doc.lisp

```diff
@@ -14,8 +14,10 @@
 
 (include-book "kestrel/utilities/gen-xdoc-for-file" :dir :system)
 
-;; Introduced by defaggregate
+;; No longer introduced by defaggregate.
+;; Now attaching to primary r1cs topic generated below with gen-xdoc-for-file.
```

Comment:
This seems incorrect.  The r1cs topic comes from books/kestrel/crypto/r1cs/doc.

Reply:
If there were no call to gen-xdoc-for-file, there would be no primary r1cs topic.  The call to gen-xdoc-for-file is below my comment in doc.lisp. I believe my comment is correct.